src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -30,17 +31,40 @@
 #include <seastar/core/memory.hh>
 #include <seastar/core/std-coroutine.hh>
 #include <seastar/http/api_docs.hh>
+#include <seastar/http/exception.hh>
 
 namespace pandaproxy::schema_registry {
 
 using server = ctx_server<service>;
 
 template<typename Handler>
 auto wrap(ss::gate& g, one_shot& os, Handler h) {
-    return [&g, &os, _h{std::move(h)}](
+    return [&g, &os, h{std::move(h)}](
              server::request_t rq,
              server::reply_t rp) -> ss::future<server::reply_t> {
-        auto h{_h};
```

Comment:
This copy is intentional `auto h{_h};`. A better alternative is now available: https://github.com/scylladb/seastar/blob/master/doc/tutorial.md#lambda-coroutines

Reply:
I was removing the `auto h{_h}` in favor of the new alternative but I can put it back.